shared/src/main/scala/drt/shared/api/WalkTime.scala

```diff
@@ -7,8 +7,8 @@ import upickle.default.{macroRW, _}
 
 import scala.collection.immutable.Map
 
-case class WalkTime(gateOrStand: String, terminal: Terminal, walkTimeMillis: Long) {
-  val inMinutes: String = millisToMinutes(walkTimeMillis)
+case class WalkTime(gateOrStand: String, terminal: Terminal, walkTimeMillis: Long, unitDisplayAsSubject: Boolean) {
```

Comment:
I don't think `unitDisplayAsSubject` belongs on this class as that means we have to know where we're going to use it for displaying text. The use as text is entirely separate from the walk time itself in my opinion
I'd say that rather we would want a helper class/object to display numbers as text

Reply:
I did bcos it already was returning "minutes" text here which was used for display so logic ended here . 

So you mean just to get the walk time minute numeral from here and display text where ever we need to display .